gosnmp.go

```diff
@@ -186,8 +186,8 @@ type SnmpPDU struct {
 	Type Asn1BER
 }
 
-// AsnExtensionID mask to identify types > 30 in subsequent byte
-const AsnExtensionID = 0x1F
+// net-snmp asn1.h ASN_OPAQUE_TAG1 (ASN_CONTEXT | ASN_EXTENSION_ID)
```

Comment:
It could be nicer to build up the ``AsnExtensionTag`` from a newly introduced constant like
```go
const AsnContext = 0x80
const AsnExtensionID = 0x1F
const AsnExtensionTag = (AsnContext | AsnExtensionID) // 0x9F
```

This way we don't have to change the public API.

Reply:
fixed

Reply:
Thank you for addressing this. Could you drop the comment line as-well, don't think we have to reference net-snmp here. Thereafter this one should be ready for final review I think.

Reply:
done